doc/source/whatsnew/v1.0.0.rst

```diff
@@ -169,6 +169,17 @@ You can use the alias ``"boolean"`` as well.
    s = pd.Series([True, False, None], dtype="boolean")
    s
 
+.. _whatsnew_1000.numba_rolling_apply:
+
+Using Numba in ``rolling.apply``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We've added an ``engine`` keyword to :meth:`~Rolling.apply` that allows the user to execute the
+routine using `Numba <https://numba.pydata.org/>`__ instead of Cython. Using the Numba engine
+can yield significant performance gains if the apply function can operate on numpy arrays and
+the data set is larger. For more details, see :ref:`rolling apply documentation <stats.rolling_apply>`
```

Comment:
"the data set is larger" here is pretty vague.  is the perf gain a function of the array size or more about the user-defined function?

Reply:
Somewhat both, but more obvious with the data size. I can make this more specific.